tests/components/ring/test_switch.py

```diff
@@ -0,0 +1,86 @@
+"""The tests for the Ring switch platform."""
+from asynctest import patch
+from homeassistant.components.switch import DOMAIN as SWITCH_DOMAIN
+from tests.common import load_fixture
+from .common import setup_platform
+
+
+async def test_entity_registry(hass, requests_mock):
+    """Tests that the devices are registed in the entity registry."""
+    with patch("ring_doorbell._exists_cache", return_value=False):
```

Comment:
Extract these patches into a pytest fixture and `yield` inside the context manager.

Reply:
I knew there would be a better way to do this. I'll see if I can find a sample.